README.md

```diff
@@ -85,13 +85,7 @@ This is a listing of example projects and guides that will help you integrate Se
 
 <details>
   <summary>Show projects</summary>
-
-  ### [webpack][28]
-  See our webpack 3 example project [here][28] (includes theming).
```

Comment:
Actually this link should be removed at all as they were removed at all (#3729).